css/style.css

```diff
@@ -160,6 +174,6 @@ aside .logo {
 
 .wix {
   background-image: url(../img/wix.jpg);
-  height: 41px;
-  margin-top: 30px;
+  height: 28px;
+  margin-top: 41px;
```

Comment:
Because `identify` == `wix.jpg JPEG 135x38 135x38+0+0 8-bit sRGB 3.12KB 0.000u 0:00.000` && `100/135*38` == `28` - 28px is the real size. Then I adjust the position with margins so that all icons are nicely vertically centered.